Scripts/Connector/Sample connector restore script.ps1

```diff
@@ -0,0 +1,34 @@
+# This script restores a connector that has been backed up.
+# To backup a Connector look at "Sample connector backup script.ps1"
+
+# RUN THIS SCRIPT IN AN ELEVATED POWERSHELL WINDOW
+
+# BEFORE RUNNING THESE COMMANDS PERFORM THE FOLLOWING STEPS:
+# 1) Copy the C:\ConnectorBackup folder (generated by the backup script) to the machine in which you will restore it
+# 2) Install the latest Connector from https://aka.ms/UPConnector (do not register it)
+ 
+# Stop services
+net stop "Print Connector service"; net stop PrintConnectorUpdaterSvc
+ 
+# Restore local printers
+c:\windows\system32\spool\tools\PrintBrm.exe -R -F C:\ConnectorBackup\printers.brm
```

Comment:
should printbrm be optional? would they typically have their printers installed already or do it separately? or would they be relying upon this script to do it?

Reply:
I don't think it should be optional, we are counting on printbrm to ensure everything else in the printers match the original one. Customers can alter the script to their specific needs if they want, but I don't think we should change it, it is more likely to lead to issues.